MAUI/DataGrid/selection.md

```diff
@@ -0,0 +1,547 @@
+When `SelectionMode` is set as Multiple, select more than one row by setting the `SfDataGrid.SelectedRows` property. 
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        //Perform selection using selected index
+        this.dataGrid.SelectedIndex = 3;
+
+        //Perform selection using selected row
+        this.dataGrid.SelectedRow = new ViewModel().OrderInfoCollection[3];
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+When the selection mode is multiple, programmatically select more than one row by adding the underlying object to be selected to the `SfDataGrid.SelectedRows` property.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        ////Perform multiple selection using selected roes
+        var viewModel = new ViewModel();
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[4]);
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[6]);
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[2]);
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[11]);
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Multiple Selection in .NET MAUI DataGrid](Images/selection/maui-datagrid-multiple-selection.png)
+
+### Select all the rows
+You can select all the rows in the datagrid using [SfDataGrid.SelectAll()] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectAll) method.
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataGrid.SelectAll();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![All the rows selected in .NET MAUI DataGrid](Images/selection/maui-datagrid-selectall.png)
+
+## Keyboard behavior
+<table>
+<tr>
+<th>
+Key or KeyCombinations
+</th>
+<th>
+Description
+</th>
+</tr>
+<tr>
+<td>
+<kbd>DownArrow</kbd>
+</td>
+<td>
+Moves CurrentCell directly below the active current cell. If the CurrentCell is in last row, pressing <kbd>DownArrow</kbd> does nothing.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>UpArrow</kbd>
+</td>
+<td>
+Moves the CurrentCell directly above the active current cell. If the CurrentCell is in first row, pressing <kbd>UpArrow</kbd> does nothing.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>LeftArrow</kbd>
+</td>
+<td>
+Moves the current cell to previous to the active current cell. If the CurrentCell is in first cell, pressing <kbd>LeftArrow</kbd> does nothing. 
+</td>
+</tr>
+<tr>
+<td>
+<kbd>RightArrow</kbd>
+</td>
+<td>
+Moves the current cell to next to the active current cell. If the CurrentCell is in last cell, pressing <kbd>RightArrow</kbd> does nothing. 
```

Comment:
Moves the current cell next to the active current cell. If the CurrentCell is in the last cell, pressing the <kbd>RightArrow</kbd> does nothing.